docs-v2/components/homepage/DesignLanguageCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Heading, Link, majorScale, Pane, Paragraph } from "evergreen-ui"
+import React from "react"
+import Thumbnail from "../Thumbnail"
+import { Item } from "../../utils/IA"
+
+interface Props {
+  img: string,
+  title: string,
+  description: string,
+  linkText?: string,
```

Comment:
Instead of passing in `linkText` and `link`, what do you think about passing in `link: JSX.Element`, so the parent component can decide what the `<Link />` should look like?

Reply:
Didn't know we can do this. Will change!

I'm using `link` as the routing path for the `Thumbnail` component though. So either we can clean up `Thumbnail`, or I'll need to pass in another prop that is called `type`.